src/sentry/search/events/fields.py

```diff
@@ -2385,9 +2385,19 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                         alias,
                     ),
                 ),
+                SnQLFunction(
+                    "eps",
+                    snql_aggregate=lambda args, alias: self._resolve_eps(args, alias),
```

Comment:
This looks redundant as `snql_aggregate=self_resolve_eps` should suffice. Additionally, is there any reason not to just inline these with the lambda? They look simple enough that we should just inline them.

Reply:
Hey Tony, good point. No particular reason not to inline. Committed this update!